### DISCUSSION TITLE
Add configurable chromakey color

### DISCUSSION THREAD

**Discussion by mishuha** [time4tea/gopro-dashboard-overlay#120] [Ideas]
Hello,

Please add the option to pass a chromakey parameter for the dashboard when generating only overlay data. The reason is that combination mode like 'lighten' in the Vegas editor completely dissolves widget black borders. Adding the chromakey (e.g pink) makes much easier any video combinations.

I'm not experienced to completely redo the script for the merging so I had only added stuff that worked for me:

layout.py:
```
class Overlay:
    def __init__(self, dimensions: Dimension, framemeta: FrameMeta, create_widgets: Callable):
        self.scene = Scene(dimensions, create_widgets(self.entry), chromakey=(255, 0, 255, 0))
        self.framemeta = framemeta
        self._entry = None
```
widgets.py:
```
class Scene:

    def __init__(self, dimensions: Dimension, widgets: List[Widget], chromakey=(0, 0, 0, 0)):
        self._widgets = widgets
        self._dimensions = dimensions
        self._chromakey = chromakey

    def draw(self) -> Image.Image:
        image = Image.new("RGBA", (self._dimensions.x, self._dimensions.y), self._chromakey)
        draw = ImageDraw.Draw(image)

        for w in self._widgets:
            w.draw(image, draw)

        return image
```
Thanks

**Comment by time4tea** [comment]
This has been released in 0.98.0. Thanks for the suggestion.

You can use it like `.... --bg 255,0,255,255`
